fmriprep/cli/parser.py

```diff
@@ -37,8 +37,7 @@ def _to_gb(value):
         return int(digits) * scale[units[0]]
 
     def _drop_sub(value):
-        value = str(value)
```

Comment:
@oesteban I'm not sure why we're coercing strings to strings, so I dropped this. Let me know if there was a subtle reason for this.

Reply:
I might have copied this from somewhere else where value could be a Path object, maybe.